CHANGELOG.md

```diff
@@ -1,3 +1,697 @@
+# 1.4.0 (2022-08-26)
+
+## Edge Agent
```

Comment:
Only the one hub change that Damon mentioned?

Reply:
Yes.